pkg/descheduler/descheduler.go

```diff
@@ -330,6 +331,8 @@ func RunDeschedulerStrategies(ctx context.Context, rs *options.DeschedulerServer
 		} else {
 			podEvictorClient = rs.Client
 		}
+		eventBroadcaster, eventRecorder := utils.GetRecorderAndBroadcaster(ctx, podEvictorClient)
```

Comment:
No need to create utils.GetRecorderAndBroadcaster in every cycle. The event recorder/broadcaster can be created only once and used throughout the entire run of the descheduler. E.g. https://github.com/kubernetes-sigs/descheduler/pull/891/commits/ca96f8387ab76c39eb26dbab091d928bdf6feaaf#diff-67816189b07b98460b57a10b41280403c53a4dfda42fdd8f54475812cb99f56eR291-R300

Reply:
Nice!